extensions/panache/mongodb-panache-common/runtime/src/main/java/io/quarkus/mongodb/panache/common/reactive/runtime/ReactiveMongoOperations.java

```diff
@@ -74,7 +76,7 @@ public Uni<Void> persist(Iterable<?> entities) {
                 objects.add(entity);
             }
 
-            if (objects.size() > 0) {
+            if (!objects.isEmpty()) {
```

Comment:
There are a ton of unrelated changes to this PR.  While I applaud code clean up, it does leave a fair bit to scroll through to get to relevant details.

Reply:
Sorry about that, I can undo these changes if needed.

Reply:
When doing such changes, it's recommended to do a separate commit (or even pull request) with the change. Given the changes are relatively safe, I think we can keep them as is.

Reply:
@gsmet sure, thanks for revision and the tip!